code/modules/food/recipes_microwave.dm

```diff
@@ -1207,3 +1205,10 @@ I said no!
 /datum/recipe/mashedpotato
 	fruit = list("potato" = 1)
 	result = /obj/item/weapon/reagent_containers/food/snacks/mashedpotato
+
+/datum/recipe/icecreamsandwich
```

Comment:
The icecreamsandwich is not listed in the changelog.
Also it seems the result is a porkbowl ?